### PR TITLE
Add youtube social icon to website

### PR DESCRIPTION
**Pull request by jjshanks**
## Description

Added youtube social icon to website.

## Motivation and Context

https://github.com/opentofu/opentofu/issues/1028

## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change depends on a change in the main [opentofu repository](https://github.com/opentofu/opentofu).

### CODE REVIEW

**Comment by janosdebugs** [comment]
Hi @jjshanks thank you for this, I'll review on Monday and get this merged.

File: docusaurus.config.js [hunk 1]
```diff
@@ -285,6 +285,13 @@ const config = {
           //     },
           //   ],
           // },
+          {
```
Comment (janosdebugs) [inline docusaurus.config.js:288]: I think this should be swapped with GitHub as it looks a bit odd:

image

**Review by janosdebugs — CHANGES_REQUESTED** [review]
Hey @jjshanks thanks a lot for this! I would like to ask for a small change.

File: docusaurus.config.js (hunk 1 above)
Comment (jjshanks) [reply docusaurus.config.js]: Good catch. Updated now
Screenshot 2024-01-08 051720

**Review by janosdebugs — APPROVED** [review]
Thank you!

**Pull request merged**